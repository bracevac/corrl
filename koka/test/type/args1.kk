struct test( x : int, y : int = 0 )

function foo() {
  Test(1)
}