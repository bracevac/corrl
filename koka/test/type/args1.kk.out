.copy : (test, x : ?int, y : ?int) -> test
.create-Test: (x : int, y : ?int) -> test
Test : (x : int, y : int) -> test
foo : () -> test
x : (test : test) -> int
y : (test : test) -> int